pkg/kudoctl/packages/package_manager.go

```diff
@@ -139,6 +133,9 @@ func packageVersionedName(pkg *PackageFiles) string {
 
 // fromFolder walks the path provided and returns CRD package files or an error
 func fromFolder(fs afero.Fs, packagePath string) (*PackageFiles, error) {
+	if packagePath == "" {
+		return nil, clog.Errorf("path must be specified")
```

Comment:
what is the advantage of using clog here? Also you're not formatting anything. Perhaps errors.New should be the right method? I always though that is the way to go when you're creating error without formatting

Reply:
I'm :+1: on standardizing on the `errors` package for everything and then using clog at the edges of our system.